pdcupdater/handlers/modules.py

```diff
@@ -17,9 +17,9 @@
 class ModuleStateChangeHandler(pdcupdater.handlers.BaseHandler):
     """ When the state of a module changes. """
 
-    tree_processing_states = set(('done', 'ready'))
+    processing_states = set(('done', 'ready', 'init')) # INIT is only for release variant creation
     other_states = set(('init', 'wait', 'building'))
```

Comment:
Hmm, now 'init' is in `(tree_)processing_states` and `other_states`... Probably get rid of this change.